wgpu-core/src/conv.rs

```diff
@@ -121,10 +122,20 @@ pub fn map_texture_usage(
         hal::TextureUses::RESOURCE,
         usage.contains(wgt::TextureUsages::TEXTURE_BINDING),
     );
-    u.set(
-        hal::TextureUses::STORAGE_READ_WRITE,
-        usage.contains(wgt::TextureUsages::STORAGE_BINDING),
-    );
+    if usage.contains(wgt::TextureUsages::STORAGE_BINDING) {
```

Comment:
`wgt::TextureUsages::STORAGE_BINDING` can't be directly translated to `hal::TextureUses` (since it has 3 variants). And we can't translate it to the union of all 3 variants since the format might not support all of them. So we use the `flags` to narrow down to the valid set of usages this textures might have.

Formats that advertise support for `STORAGE_BINDING` must advertise support for one of the access modes.